smarts/core/smarts.py

```diff
@@ -1385,13 +1385,21 @@ def _try_emit_envision_state(self, provider_state: ProviderState, obs, scores):
                 veh_type = (
                     v.vehicle_config_type if v.vehicle_config_type else v.vehicle_type
                 )
-                traffic[v.vehicle_id] = envision_types.TrafficActorState(
-                    actor_type=envision_types.TrafficActorType.SocialVehicle,
-                    vehicle_type=veh_type,
-                    position=tuple(v.pose.position),
-                    heading=float(v.pose.heading),
-                    speed=v.speed,
-                )
+
+                if veh_type == "passenger":
+                    veh_type = "car"
+
+                SUPPORTED_TYPES = set(item.value for item in envision_types.VehicleType)
+
+                if veh_type in SUPPORTED_TYPES:
```

Comment:
Maybe we should log a warning message if `veh_type` is not in the `SUPPORTED_TYPES` list?  Something like `"not sending vehicle {v.vehicle_id} to Envision since it is of unsupported type {veh_type}"`...

Reply:
... but if so, it probably only needs to be logged once per `v.vehicle_id`, so I guess this is optional.